src/x/instrument/build.go

```diff
@@ -90,6 +96,24 @@ func LogBuildInfoWithLogger(logger *log.Logger) {
 	logger.Printf("Build TimeUnix:     %s\n", BuildTimeUnix)
 }
 
+// LogBuildInfoWithLogger logs the build information using the provided logger in JSON
+func LogBuildInfoWithLoggerJSON(logger *log.Logger) {
+	buildMap := make(map[string]string)
+	buildMap["go_runtime_version"] = goVersion
+	buildMap["build_version"] = Version
+	buildMap["build_revision"] = Revision
+	buildMap["build_branch"] = Branch
+	buildMap["build_date"] = BuildDate
+	buildMap["build_time_unix"] = BuildTimeUnix
+
+	jsonOut, _ := json.Marshal(buildMap)
```

Comment:
Please do not ignore the error here. We want to handle this because it could return bogus data and potentially panic.

Reply:
on that note could we add a dumb test case here as a regression test?